index.html

```diff
@@ -3,20 +3,54 @@
   <head>
     <meta charset="utf-8">
     <title>ACLU Cash Bail</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/transition.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/dropdown.min.css">
     <link rel="stylesheet" href="src/css/style.css">
     <link rel="stylesheet" href="src/css/table.css">
+    <link rel="stylesheet" href="src/css/search.css">
   </head>
   <body>
     <div class="container">
-      <table id="bail-rate-table">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
-      <table id="bail-race-table">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
-    </div>
-    <script src="src/js/index.js" charset="utf-8"></script>
+      <div id="bail-rate-container">
+        <div class="ui fluid multiple search selection dropdown">
+          <input type="hidden" name="county">
+          <i class="dropdown icon"></i>
+          <div class="default text">Select County</div>
```

Comment:
This should be "Select Counties" per the designs.

Reply:
Done